dashboard-ui/src/pages/home/log-metadata-provider.tsx

```diff
@@ -155,9 +161,9 @@ export const LogMetadataProvider = ({ kubeContext }: LogMetadataProviderProps) =
     return subscribeToMore({
       document: LOG_METADATA_LIST_WATCH,
       updateQuery: (prev, { subscriptionData }) => {
-        const ev = subscriptionData.data.logMetadataWatch;
+        const ev = (subscriptionData.data as any)?.logMetadataWatch;
         if (ev) eventBufferRef.current.push(ev);
-        return prev;
+        return prev as any;
```

Comment:
It looks like you need a type cast here because `prev` has type `DeepPartialObject<T>` but `updateQuery()` expects `T`. We can revisit this later but for now let's typecast as `T`:
```suggestion
        return prev as LogMetadataListFetchQuery;
```

You'll also need to import the type object in the beginning of the file:
```typescript
import type { LogMetadataListFetchQuery, LogMetadataWatchEvent } from '@/lib/graphql/cluster-api/__generated__/graphql';
```